src/background/responders.ts

```diff
@@ -131,10 +109,11 @@ export const responders: Responders<Action> = {
     nextTabs.set(tab.id, {
       ...tab,
       feedbackState: {
```

Comment:
I think we can call `updateActiveFeedback` every time we update the feedback state